src/views/officer_seattle.j2

```diff
@@ -8,4 +8,4 @@
 <p><b>Hourly Rate:</b> ${{ hourly_rate }}</p>
 <p><b>Projected salary (w/o overtime):</b> ${{ projected }}</p>
 {% endif %}
-<p><i>Record Source Roster Date: {{ date }}</i></p>
+<p><i>Record Source Roster Date*: {{ date }}</i></p>
```

Comment:
I think the star should be at the beginning of these lines to denote that this is the field describing the starred attributes